plugins/org.jkiss.dbeaver.ext.generic.ui/src/org/jkiss/dbeaver/ext/generic/views/GenericConnectionPage.java

```diff
@@ -250,6 +256,10 @@ public void widgetSelected(SelectionEvent e) {
         setControl(addrGroup);
     }
 
+    protected void addAdditionalGeneralControls(Composite settingsGroup) {
```

Comment:
Are you sure that you can't use GenericConnectionPage#createAdvancedSettingsGroup?

Reply:
It will be hanging on the end of the page, will not look good at all

Reply:
It looks ok for Teradata.

![2022-09-07 14_24_37-Connect to a database](https://user-images.githubusercontent.com/45152336/188867119-fe76c6b9-2480-4545-ae1c-41585225c279.png)

Protocol settings in both cases.

It will be inconsistent.